.gitignore

```diff
@@ -54,5 +54,3 @@ report
 *.vim
 test.html
 
-# Compiled files
-dist
```

Comment:
We managing bundle files by `production` branch.
I think you must not remove this line.